### PR TITLE
548115 - [Passage] provide ability to create product version from issue license wizard

### PR DESCRIPTION
**Pull request by ruspl-afed**
Fixed several annoying issues for "select or create" scenario:
* shell overlap for nesting wizards
* label for container field
* dialog and shell titles

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizard.java [hunk 1]
```diff
@@ -73,13 +74,16 @@ protected void store(Optional<R> container, EObject candidate) {
...
-				.get();
+		EReference reference = containerEReference(candidate.eClass()).get();
 		candidate.eSet(reference, container.get());
 		Resource resource = candidate.eResource();
 		Optional.ofNullable(resource).ifPresent(r -> LocWokbench.save(r));
 	}
 
+	private Optional<EReference> containerEReference(EClass eClass) {
+		return eClass.getEAllReferences().stream()//
+				.filter(r -> r.isContainer())//
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizard.java:85]: static reference `EReference::isContainer` should work

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizardPage.java [hunk 2]
```diff
@@ -45,18 +51,17 @@ protected InnerClassifierWizardPage(EntityMetadata metadata, ClassifierInitializ
...
 		this.context = context;
+		this.forward = new MandatoryEclipseContext(EclipseContextFactory
+				.createServiceContext(FrameworkUtil.getBundle(getClass()).getBundleContext()).createChild());
 	}
 
 	@Override
 	protected void createFieldControls(Composite composite) {
-		text = createTextButtonBlock(composite, labelForContainer(), () -> selectContainer());
+		text = createTextButtonBlock(composite, new EClassName(containerMetadata().get().eClass()).get(),
+				() -> selectContainer());
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizardPage.java:61]: `this::seletContainer`?

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/moveto/lic/internal/features/model/FeaturesClassMetadata.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+				new PlainEntityMetadata(//
+						meta.getFeatureSet(), //
+						meta.getFeatureSet_Identifier(), //
+						meta.getFeatureSet_Name()));
+		map.put(FeatureDescriptor.class, //
+				new PlainEntityMetadata(//
+						meta.getFeature(), //
+						meta.getFeature_Identifier(), //
+						meta.getFeature_Name()));
+	}
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/moveto/lic/internal/features/model/FeaturesClassMetadata.java:46]: Do we anyway need these data after any `FeaturesClassMetadata` construction or should we use something `Cached` instead?

Same question for `LicensesClassMetadata`.

Comment (ruspl-afed) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/moveto/lic/internal/features/model/FeaturesClassMetadata.java:46]: That may be a good task for @zelenyhleb

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizard.java (hunk 1 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizard.java]: done

File: bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizardPage.java (hunk 2 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizardPage.java]: done

**Comment by ruspl-afed** [comment]
@eparovyshnaya please have a look, I fixed all the labels for nested wizards

File: bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/moveto/lic/internal/features/model/FeaturesClassMetadata.java (hunk 3 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/moveto/lic/internal/features/model/FeaturesClassMetadata.java:46]: Agree

**Pull request merged**